### PR TITLE
Add container path on dynaconf setting

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

**Comment by fao89** [comment]
Tested here: 
https://github.com/pulp/pulp_ansible/pull/1089

**Review by rochacbruno — APPROVED** [review]

**Comment by bmbouter** [comment]
I meant to get to this earlier, but there is so much going on. What's the motivation for this?

**Comment by fao89** [comment]
> I meant to get to this earlier, but there is so much going on. What's the motivation for this?

allowing older branches to have functional tests working inside the container

**Comment by bmbouter** [comment]
Sounds good. How does this solve it? I'm unclear on what the root cause preventing it was.

File: pulp_ansible/tests/functional/utils.py [hunk 1]
```diff
@@ -322,7 +322,11 @@ def _create_empty_repo_and_distribution(self, cleanup=True):
 
 
 settings = Dynaconf(
-    settings_files=["pulp_ansible/tests/assets/func_test_settings.py", "/etc/pulp/settings.py"]
+    settings_files=[
+        "pulp_ansible/tests/assets/func_test_settings.py",
```
Comment (fao89) [inline pulp_ansible/tests/functional/utils.py:326]: the tests couldn't get this, so it didn't load DATABASE setting:
https://github.com/pulp/pulp_ansible/runs/7172035382?check_suite_focus=true#step:11:1493

**Pull request merged**